### ISSUE TITLE
result return for postprocess function.

### ISSUE THREAD

**Issue by Zheweiqiu** [zylo117/Yet-Another-EfficientDet-Pytorch#374]
Hi,
I am back:). I just notice there is problem when feeding model with batch of images.
I did an example, I passes two images for each inference (batch_size=2), here is what I got for result:
['/home/qiuzhewei/val/89f46599053d4b42bb9d3353844037d2_1.jpg', '/home/qiuzhewei/val/b2cfe0c19b004448a6df0f9c441c7848_0.jpg'] 
[{'rois': array([], dtype=float64), 'class_ids': array([], dtype=float64), 'scores': array([], dtype=float64)}, {'rois': array([], dtype=float64), 'class_ids': array([], dtype=float64), 'scores': array([], dtype=float64)}, {'rois': array([], dtype=float64), 'class_ids': array([], dtype=float64), 'scores': array([], dtype=float64)}, {'rois': array([], dtype=float64), 'class_ids': array([], dtype=float64), 'scores': array([], dtype=float64)}]

The first list contains path of two images and the second one contains four detecting results, but from the results, it is impossible to tell which results are for the first image and which are for the second.
I think the result should be embedded with another [], like something below:

[[{'rois': array([], dtype=float64), 'class_ids': array([], dtype=float64), 'scores': array([], dtype=float64)}, {'rois': array([], dtype=float64), 'class_ids': array([], dtype=float64), 'scores': array([], dtype=float64)}], [{'rois': array([], dtype=float64), 'class_ids': array([], dtype=float64), 'scores': array([], dtype=float64)}, {'rois': array([], dtype=float64), 'class_ids': array([], dtype=float64), 'scores': array([], dtype=float64)}]]

I found the problem lies on postprocess function when I trace back step by step.

Could you please check that? I might be wrong.

Best,

**Comment by zylo117** [comment]
https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/blob/master/utils/utils.py#L97
the result is generated for every image

**Comment by Zheweiqiu** [comment]
> https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/blob/master/utils/utils.py#L97
> the result is generated for every image

Right, but the results are squeezed in one single list and we cannot tell that which part of results are generated by which image.

**Comment by zylo117** [comment]
you might want to update the code for there was a bug and try again
